media/js/custom.js

```diff
@@ -98,3 +98,9 @@ $(function() {
         infinite: true
     });
 });
+
+$('.teachers-alphabet ul li').click(function () {
+	$('.teachers-container div').css('display','none');
+	var thisItem = $(this).attr('class');
+	$('.teachers-'+thisItem).css('display','block');
+});
```

Comment:
No newline at end of file. Have you tried turning on the setting to add a newline at end of file on save?